### PR TITLE
Increase covd coverage for paramtable

### PR DESCRIPTION
**Pull request by xiaocai2333**
Signed-off-by: cai.zhang <cai.zhang@zilliz.com>
issue: #7476

### CODE REVIEW

File: internal/indexcoord/paramtable.go [hunk 1]
```diff
@@ -61,6 +62,20 @@ func (pt *ParamTable) Init() {
 	})
 }
 
+func (pt *ParamTable) initAddress() {
+	ret, err := pt.Load("_IndexCoordAddress")
+	log.Debug("err", zap.Any("err", err))
+	if err != nil {
+		log.Debug("haha")
```
Comment (czs007) [inline internal/indexcoord/paramtable.go:69]: what's. this?

File: internal/kv/mem/mem_kv.go [hunk 2]
```diff
@@ -44,7 +45,7 @@ func (kv *MemoryKV) Load(key string) (string, error) {
 	item := kv.tree.Get(memoryKVItem{key, ""})
 	// TODO，load unexisted key behavior is weird
 	if item == nil {
-		return "", nil
+		return "", fmt.Errorf("key %s is not exist", key)
```
Comment (czs007) [inline internal/kv/mem/mem_kv.go:48]: reject for now. As the config items used in helm are not the same as YAML files located in the configs directory.

File: internal/indexcoord/paramtable.go (hunk 1 above)
Comment (xiaocai2333) [reply internal/indexcoord/paramtable.go]: Unnecessary log, have deleted.

File: internal/kv/mem/mem_kv.go (hunk 2 above)
Comment (xiaocai2333) [reply internal/kv/mem/mem_kv.go]: Yep.

File: internal/indexcoord/paramtable.go [hunk 3]
```diff
@@ -61,6 +60,18 @@ func (pt *ParamTable) Init() {
 	})
 }
 
+func (pt *ParamTable) initAddress() {
```
Comment (czs007) [inline internal/indexcoord/paramtable.go:63]: please delete this. unused for now.

File: internal/indexnode/index.go [hunk 4]
```diff
@@ -56,13 +56,13 @@ func (index *CIndex) Serialize() ([]*Blob, error) {
 	var cBinary C.CBinary
 
 	status := C.SerializeToSlicedBuffer(index.indexPtr, &cBinary)
-	defer C.DeleteCBinary(cBinary)
```
Comment (czs007) [inline internal/indexnode/index.go:59]: another soluation
defer func(){
   if C.CBinary is empty{
     return 
  }
 C.DeleteBinary(cBinary)
}

**Pull request merged**